### ISSUE TITLE
Duty plugin

### ISSUE THREAD

**Issue by marek-saji** [marek-saji/suchbot#34]
Come up with broader name.

Blocked by #33, #8. Should share lots of code with `lunches` plugin.

Example config:

```json
{
    "spreadsheetId": "…",
    "dateRange": "…",
    "nameRange": "…",
    "time": "8:00",
    "message": "You are on cleaning duty today."
}
```

Optional features:

- additional message by cell’s background colour

**Comment by marek-saji** [comment]
Duplicate of #15.
